src/main/java/duke/tasks/Deadline.java

```diff
@@ -0,0 +1,44 @@
+package duke.tasks;
+import java.time.*;
+import java.time.format.*;
+
+public class Deadline extends Task {
+    private LocalDateTime deadline;
+
+    //Constructor
+    public Deadline (String name, LocalDateTime deadline) {
```

Comment:
Should there be a space between the class name and the argument bracket?
```suggestion
    public Deadline(String name, LocalDateTime deadline) {
```